AutonomousDemo/Velocity.cs

```diff
@@ -9,6 +9,10 @@ namespace AutonomousDemo
     public class Velocity
     {
         public double Speed { get; set; } = 0;
+        /// <summary>
+        /// 
+        /// Direction unit is radians
```

Comment:
Remove extra space. Also this is what displays as a mouseover comment if you hover over a particular property (or method or whatever), so picture yourself as another developer thinking, "What actually is direction?" and then put the answer here (e.g., "Direction on a two-dimensional surface, in radians. Assumes that 0 radians is due East." or something).